src/state/appSessions/index.ts

```diff
@@ -1,8 +1,23 @@
 import { Address } from 'viem';
 
-import { Network, ChainId, networkToIdMapping } from '@/networks/types';
+import { Network, ChainId } from '@/chains/types';
 import { createRainbowStore } from '../internal/createRainbowStore';
 
+const chainsIdByNetwork: Record<Network, ChainId> = {
```

Comment:
did you want this replaced by `chainsIdByName` from `@/chains` instead?

Reply:
I did it on purpose

For some reason this was causing some issues in unit tests do i had to use the old Network type here to fix it